CTP/conf/shell.conf

```diff
@@ -79,8 +79,8 @@ main.testing.category=shell
 # To skip the case which is not compatibility with the different platform
 main.testcase.skip_key=LINUX_NOT_SUPPORTED
 
-# The max retry count when the case fail
+# The max retry count when the case fail, if the number of success is more than half of the max retry count, it will be considered as PASS
```

Comment:
"if the number of success is more than half of the max retry count, it will be considered as PASS"

We need confirm it from codes.